Package.swift

```diff
@@ -15,7 +15,7 @@ let package = Package(
             targets: ["UID2GMAPlugin"])
     ],
     dependencies: [
-        .package(url: "https://github.com/IABTechLab/uid2-ios-sdk.git", from: "0.1.0"),
+        .package(url: "https://github.com/IABTechLab/uid2-ios-sdk.git", from: "0.2.0"),
```

Comment:
This is incorrect as the UID2 iOS SDK does not have a v0.2.0 published yet, only v0.1.0. We could do a v0.2.0 release of the SDK though as a few things have changed since v0.1.0 was released on 31-March-2023. It might also make things easier to track if the SDK and the Plugins all have the same version number. What do you think?

BTW... the SDK Versions can be tracked here -->

https://github.com/IABTechLab/uid2-ios-sdk/releases

Reply:
SDK v0.2.0 has been released.